config/prow/k8s/app/svc-envoy.yaml

```diff
@@ -1,23 +0,0 @@
-apiVersion: v1
-kind: Service
-metadata:
-  annotations:
-    service.beta.kubernetes.io/azure-load-balancer-resource-group: $AZURE_RG
-    service.beta.kubernetes.io/azure-pip-name: $PUBLIC_IP_NAME
-  name: envoy
-  namespace: projectcontour
-spec:
-  externalTrafficPolicy: Local
-  ports:
-  - port: 80
-    name: http
-    protocol: TCP
-    targetPort: 8080
-  - port: 443
-    name: https
-    protocol: TCP
-    targetPort: 8443
-  selector:
-    app: envoy
-  type: LoadBalancer
-  loadBalancerIP: $PUBLIC_IP_ADDRESS
```

Comment:
"loadBalancerIP: $PUBLIC_IP_ADDRESS" is missing in new edit in `contour.yaml`, but I guess it is ok if the branch is successfully deployed?

Reply:
@haitch  would have another PR to address this .

Reply:
the IP will be solved by cloud-provider, the issue I am fixing is permission for cloud-provider.